CHANGELOG

```diff
@@ -7,6 +7,8 @@ https://openslides.org/
 Version 2.1.2 (unreleased)
 ==========================
 
+- Fixed issue when creating/deleting motion comment fields in the settings.
```

Comment:
Please change "2.1.2" to "2.2" in CHANGELOG. All new pull requests are scheduled for 2.2...